gobblin-core/src/main/java/gobblin/metrics/GobblinMetrics.java

```diff
@@ -314,8 +333,14 @@ private void buildFileMetricReporter(Properties properties) {
         return;
       }
 
+      // Add a suffix to file name if specified in properties.
+      String metricsFileSuffix = properties.getProperty(ConfigurationKeys.METRICS_FILE_SUFFIX, "");
```

Comment:
Since you check `ifNullOrEmpty` on `metricsFileSuffix`, there's no need to provide a default of empty string here.